packages/protocol/contracts/stability/test/MockStableToken.sol

```diff
@@ -3,64 +3,78 @@ pragma solidity ^0.5.13;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
+import "../../common/FixidityLib.sol";
+
 /**
  * @title A mock StableToken for testing.
  */
 contract MockStableToken {
+  using FixidityLib for FixidityLib.Fraction;
   using SafeMath for uint256;
 
   uint8 public constant decimals = 18;
-  bool public _needsRebase;
   uint256 public _totalSupply;
-  uint256 public _targetTotalSupply;
-  mapping(address => uint256) public balanceOf;
+  FixidityLib.Fraction public inflationFactor;
+
+  // Stored as units. Value can be found using unitsToValue().
+  mapping(address => uint256) public balances;
 
-  function setNeedsRebase() external {
-    _needsRebase = true;
+  constructor() public {
+    setInflationFactor(FixidityLib.fixed1().unwrap());
```

Comment:
not specific to this pr, but an "inflation factor of 1" suggests 100% inflation, not 0